src/index.js

```diff
@@ -2,31 +2,61 @@
 import './styles.css';
 import { renderListZone, renderList } from './modules/leaderboardList';
 import renderFormZone from './modules/leaderboardForm';
+import Store from './modules/store';
+import leaderAPI from './modules/leaderAPI';
+import validateScore from './modules/helpfulFunctions';
 
-// Hardcoded scores
-const scoreList = [];
-const score1 = {};
-score1.name = 'John';
-score1.points = 100;
-scoreList.push(score1);
-const score2 = {};
-score2.name = 'Jane';
-score2.points = 333;
-scoreList.push(score2);
-const score3 = {};
-score3.name = 'Joe';
-score3.points = 7;
-scoreList.push(score3);
-const score4 = {};
-score4.name = 'Jenny';
-score4.points = 55;
-scoreList.push(score4);
+// variables
+let scoreList = [];
+let gameKey;
+const newGameURL = `
+  https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/`;
+const newGameName = { name: "Elmer's Glue" };
 
-const displayStuff = () => {
-  renderListZone();
-  renderList(scoreList);
+gameKey = Store.getGameKey();
+
+const iniateLeaderboard = () => {
   renderFormZone();
+  renderListZone();
+  if (!gameKey) { // NO gameKey in localStorage
+    leaderAPI.postName(newGameName, newGameURL).then((res) => {
+      Store.setGameKey(res);
+      gameKey = res;
+      leaderAPI.getScores(newGameURL, gameKey).then((res) => {
+        scoreList = res.result;
+        renderList(scoreList);
+      });
+    });
+  } else { // gameKey is in localStorage
+    leaderAPI.getScores(newGameURL, gameKey).then((res) => {
+      scoreList = res.result;
+      renderList(scoreList);
+    });
```

Comment:
- [ ] Hello the functionality of your code works correctly, although for this project you are required to use `async\await` syntax rather than using the `.then` method to handle the asynchronous activity. please make this change and your project should be complete